_pages/en/strategy-target-solution-delivery-model.md

```diff
@@ -625,75 +667,71 @@ The metrics are defined as follows:
+In addition, the software world is moving away from [Monolith-types of solutions](https://medium.com/koderlabs/introduction-to-monolithic-architecture-and-microservices-architecture-b211a5955c63) (large code bases that provides many capabilities but become bottlenecks and single-points of failures in the IT ecosystem as the many IT teams needed to perform work collide).
 
-In the House of Commons questions data, ESDC has 52 projects listed but only 6 of them have sufficient data to compare either their schedule or budget estimates: 5 are behind schedule, and 5 are over budget.
+The [TB Directive on the Management of Projects and Programmes](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32594) makes room for the above approach:
 
-The [2019-2020 February (P11) Monthly Project and Programme Investment Portfolio Dashboard](http://dialogue/grp/IPPST/Financials%20Limited%20Access/Monthly%20Project%20Portfolio%20Dashboard%20-%20January%202019-2020%20(P10).xlsx) of ESDC's EPMO  lists 53 active projects.
-45% of them are deemed healthy (green), 19% are deemed of concerns (yellow), and 34% are unhealthy (red).
-Looking deeper, it's worth noting the following:
-    - that 38 of the 53 active projects are in the Execution stage and that 55% of them are either of concerns (yellow) or unhealthy (red)
-    - that, generally, the IITB component of those ESDC projects are healthier than the overall Departmental project umbrella
+- [The Project Sponsor is responsible for:] 4.2.5 Maintaining effective relationships with key external stakeholders including implicated departments and common service providers
+- [The Project Sponsor is responsible for:] 4.2.8 Applying as appropriate, incremental, iterative, agile, and user-centric principles and methods to the planning, definition, and implementation of the project
+- [The Project Sponsor is responsible for:] 4.2.18 Establishing a project gating plan at the outset of the project, consistent with the department's framework, that [4.2.18.1] Documents the decisions that will be taken at each gate, the evidence and information required in support of the gate decisions, the criteria used to assess the evidence, and the gate governance
 
-IITB’s [BPMO Project Portfolio dashboard](http://dialogue/grp/ITPM-GPTI/PPDMLibrary/BRM-RPT-PDM-ProjectPortfolioDashboard20200217.xlsm) also tracks lite projects, branch initiatives, and A-based investments (smaller in size than the ones tracked by the EPMO).
-The February 2020 statistics show a total of 26 active projects of this category.
-69% of them are deemed healthy (green), 15% are deemed of concerns (yellow), and 15% are deemed unhealthy (red).
+The above 3 requirements from the TB Directive indicates that departments must accept and adapt to change, as well as make evidence-based decision making (like planning decisions). Such evidence can only be obtained by execution.
```

Comment:
Just because of the other comments, I would suggest changing the 3 to three

Also..."...as well as make evidence-based decisions" (rather than decision making)